### PR TITLE
Main

### PR DESCRIPTION
**Pull request by Manikanta1432**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

**Comment by Manibz** [comment]
Please merge if there are no errors

**Comment by Manikanta1432** [comment]
please merge if there are no errors

File: solutions/cses-1662.mdx [hunk 1]
```diff
@@ -128,4 +128,31 @@ public class subarrayDivisibility {
 ```
 </JavaSection>
 	
-</LanguageSection>
+<PySection>
+
+```py 	
+
```
Comment (Aadit-Ambadkar) [inline solutions/cses-1662.mdx:134]: fix your spacing according to the guidelines. There is unnecessary spacing.

File: solutions/cses-1662.mdx [hunk 2]
```diff
@@ -128,4 +128,31 @@ public class subarrayDivisibility {
...
+	# Remember to account for negative sums
+	
+	temp_Array [ ( psums % n + n ) % n ] += 1
+ans = 0	
+for x in temp_Array:
+       
+       #Calculating the # of pairs.
+	 # This calculates the pairs without
+	 # duplicates and reverse groups.
+	ans += x * (x - 1) //2
```
Comment (Aadit-Ambadkar) [inline solutions/cses-1662.mdx:152]:
```suggestion
	ans += x * (x - 1) // 2
```

File: solutions/cses-1662.mdx (hunk 1 above)
Comment (Aadit-Ambadkar) [inline solutions/cses-1662.mdx:134]:
```suggestion
```

File: solutions/cses-1662.mdx [hunk 3]
```diff
@@ -128,4 +128,31 @@ public class subarrayDivisibility {
...
+for i in input_Array:
+	psums += i
+	
+	# Remember to account for negative sums
+	
+	temp_Array [ ( psums % n + n ) % n ] += 1
+ans = 0	
+for x in temp_Array:
+       
+       #Calculating the # of pairs.
```
Comment (Aadit-Ambadkar) [inline solutions/cses-1662.mdx:149]:
```suggestion
        #Calculating the # of pairs.
```

File: solutions/cses-1662.mdx (hunk 1 above)
Comment (Aadit-Ambadkar) [reply solutions/cses-1662.mdx:134]: also tabs, not spaces!

**Review by Aadit-Ambadkar — CHANGES_REQUESTED** [review]
lots to fix

**Comment by Aadit-Ambadkar** [comment]
> please merge if there are no errors

lol be patient there wait for at least 3 days before bumping

**Pull request closed without merging**